packages/amplication-client/src/Workspaces/ResourceListItem.scss

```diff
@@ -26,16 +26,20 @@
     height: var(--medium-button-height);
     border-radius: var(--medium-button-radius);
     @include flexFullRow;
-    color: var(--black100);
     box-sizing: border-box;
     white-space: nowrap;
-
     .user-and-time {
       margin-left: var(--default-spacing-small);
     }
   }
 
   &__last-build {
     font-weight: normal;
+    &__title {
+      color: var(--black100);
```

Comment:
Why did you add it? you already have it (line 15)
![image](https://user-images.githubusercontent.com/39680385/187164255-8f0c90fb-6bf3-41cc-8f67-5d4815cc172d.png)

Reply:
It's a different type of title, it's the panel title.